### DISCUSSION TITLE
New sidepanel features

### DISCUSSION THREAD

**Discussion by Sci-Mon** [KDE/ghostwriter#859] [Ideas]
* Dialog to configure font and fontsize of the side-panel
* capability of folding for the outline display similar to  acrobat-reader bookmarks or file explorers with a folding-tree

**Comment by patrix67** [comment]
Please, where can I edit the help side-panel (_to add for example the code for footnotes_).
Thanks for response,

**Reply by Sci-Mon** [reply]
Seems to be placed/defined in the `mainwindow.cpp`...
No easy configuration without re-compiling the sourceode

void MainWindow::setupSidebar()
{
    cheatSheetWidget = new QListWidget(this);

    cheatSheetWidget->setSelectionMode(QAbstractItemView::NoSelection);
    cheatSheetWidget->setAlternatingRowColors(false);

    cheatSheetWidget->addItem(tr("# Heading 1"));
    cheatSheetWidget->addItem(tr("## Heading 2"));
    cheatSheetWidget->addItem(tr("### Heading 3"));
    cheatSheetWidget->addItem(tr("#### Heading 4"));
    cheatSheetWidget->addItem(tr("##### Heading 5"));
    cheatSheetWidget->addItem(tr("###### Heading 6"));
